cmd/lakefs/cmd/import.go

```diff
@@ -125,7 +125,8 @@ func runImport(cmd *cobra.Command, args []string) (statusCode int) {
 	var idGen actions.IDGenerator
 	var actionsStore actions.Store
 	if dbParams.KVEnabled {
-		kvStore, err := kv.Open(ctx, dbParams.Type, dbParams.ConnectionString)
+		kvparams := cfg.GetKVParams()
+		kvStore, err := kv.Open(ctx, dbParams.Type, kvparams)
```

Comment:
Just a thought:
How about the option to send the entire `cfg` to `kv.Open` and it perfrom the `cfg.GetKVParams()`?
On a logical level it means that KV is "aware" of the entire `config` package and that it (`cfg`) has a designated section (`kvparams`) for its ('KV') use, which makes sense, and that the `cmd` package, does not need to be aware of `kvparams` which also makes sense (especially if and when `kvparams` will change)

Reply:
I think it's better that the wiring happens and a single place. Also `kv` doesn't have a dependency on the `config` pkg, it does have a dependency on its inner `params` package. That's the same pattern the `db` package uses and without clear advantages to any solution, I rather keep it consistent.